content/en/docs/setup/install/operator/index.md

```diff
@@ -68,27 +68,6 @@ $ istioctl operator init --watchedNamespaces=istio-namespace1,istio-namespace2
 
 See the [`istioctl operator init` command reference](/docs/reference/commands/istioctl/#istioctl-operator-init) for details.
 
-{{< tip >}}
-You can alternatively deploy the operator using Helm:
-
-1. Create a namespace `istio-operator`.
-
-    {{< text bash >}}
-    $ kubectl create namespace istio-operator
-    {{< /text >}}
-
-1. Install operator using Helm.
-
-    {{< text bash >}}
-    $ helm install istio-operator manifests/charts/istio-operator \
-        --set watchedNamespaces="istio-namespace1\,istio-namespace2" \
-        -n istio-operator
-    {{< /text >}}
-
-Note that you need to [download the Istio release](/docs/setup/getting-started/#download)
-to run the above command.
-{{< /tip >}}
-
 {{< warning >}}
```

Comment:
Also remove the warning? Talks about creating the namespace which is being removed in the tip.

Reply:
will remove

Reply:
Actually, re-reading this, the `init` is what creates the namespace, and you are not updating that, so you could leave the tip. It talks about differences from 1.10 which is no longer supported, so removing it is fine as well.